src/privileged/trackers/api.js

```diff
@@ -31,8 +31,8 @@ class TrackersEventEmitter extends EventEmitter {
   emitErrorDetected(error, tabId) {
     this.emit("page-error-detected", error, tabId);
   }
-  emitAddException(tabId) {
-    this.emit("exception-added", tabId);
+  emitToggleException(tabId, toggleValue) {
+    this.emit("exception-added", tabId, toggleValue);
```

Comment:
nit: should this event name be `exception-toggled` instead of `exception-added` now?

Reply:
oh yes, absolutely!